README.md

```diff
@@ -170,7 +170,9 @@ To allow loading pages from the old code in an iframe (which is necessary for tu
 
 The project should now build. Confirm this via *Build* → *Build Solution* (or `CTRL+SHIFT+B`).
 
-You can now run the app by clicking the play button (▶), which should say *IIS Express*.
+You can now run the app:
+* In Visual Studio, clicking the play button (▶), which should say *IIS Express*.
+* In Rider, select *DigitalLearningSolutions.Web: IIS Express* from the Run Configurations dropdown, and then click the Run button.

```

Comment:
I would just make the grammar consistent here, like maybe change "clicking" to "click" to match the "select" in the second option